### ISSUE TITLE
Exception has been thrown by the target of an invocation.

### ISSUE THREAD

**Issue by alyxaugustine** [DurtyFree/durty-cloth-tool#353]
I have the following error:
```
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.OverflowException: Arithmetic operation resulted in an overflow.
   at CodeWalker.GameFiles.ResourceBuilder.Build(ResourceFileBase fileBase, Int32 version, Boolean compress)
   at CodeWalker.GameFiles.YtdFile.Save()
   at hc3tEphwfAaqudJWjZGj.c9ysrR0Ya75(Object , hc3tEphwfAaqudJWjZGj )
   at wAqVg37ayUV4FNj8DAK.g9edYV7p4QLSwqL7njj.Ily7UFN9Sl(ClothData  , String[]  )
   at wAqVg37ayUV4FNj8DAK.g9edYV7p4QLSwqL7njj.RHp78waicQ(ClothData  )
   at rQmql1uCtDYM7E8xIY4.gFe59quMjtCH6qNFgyr.n879YhhryvHfqRDHSlxM(Object , Object )
   at rQmql1uCtDYM7E8xIY4.gFe59quMjtCH6qNFgyr.roUue0UsxQ(Object  , RoutedEventArgs  )
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Delegate.DynamicInvokeImpl(Object[] args)
   at System.Windows.RoutedEventArgs.InvokeEventHandler(Delegate genericHandler, Object genericTarget)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at Telerik.Windows.Controls.RadMenuItem.OnClickImpl()
   at Telerik.Windows.Controls.RadMenuItem.HandleMouseUp()
   at Telerik.Windows.Controls.RadMenuItem.OnMouseLeftButtonUp(MouseButtonEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.ReRaiseEventAs(DependencyObject sender, RoutedEventArgs args, RoutedEvent newEvent)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   at System.Windows.Input.InputManager.ProcessStagingArea()
   at System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   at System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
   at System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

App Version: **3.0.0.8**
Project targets: **FiveM**
DLC name: **egrpcloth**
Collection name: **egrpcloth // **
Project path: **D:\FIVEM DEV\DurtyClothTool.App\JoaatOutput\clothproject_egrpcloth**
Output path: **D:\FIVEM DEV\DurtyClothTool.App\JoaatOutput**
Disable Extra Ymt Creation: **True**
Create Subfolder For Cloth Drawable Types: **True**
Replace settings enabled: **False // DLC: NoDlc**
Save File Version: **5**
```

Cloth files:
`[Please provide cloth files (ydd, ytd), cloth project and any other related files here + describe reproduction steps]`

**Comment by DurtyFree** [comment]
Fixed next update

**Comment by DurtyFree** [comment]
Fixed in v3.1 https://github.com/DurtyFree/durty-cloth-tool/releases/tag/3.1.0.0
